docs/run-core-node/monitoring.mdx

```diff
@@ -317,7 +317,7 @@ Installing and configuring a Prometheus server is out of scope of this document,
 
 ### Install the stellar-core-prometheus-exporter
 
-The stellar-core-prometheus-exporter is an exporter that scrapes the `stellar-core` metrics endpoint (`http://localhost:11626/metrics`) and renders these metrics in the Prometheus text-based format available for Prometheus to scrape and store in its timeseries database.
+The stellar-core-prometheus-exporter is an exporter that scrapes the `stellar-core` metrics endpoint (`http://localhost:11626/metrics`) and renders these metrics in the Prometheus text-based format available for Prometheus to scrape and store in its time series database.
```

Comment:
@ElliotFriend what do you think of this one?

Reply:
> @tyvdh can you see why this is failing the MDX linting check aka the bane of my existence?

i fixed that linting check in [65] it was just some small things in code examples. 

In the future if you start getting failing checks run `npm run format:mdx` and it should pretty print.

The other warning about the deprecated command shouldn't cause failures.

Reply:
`time series` seems to be the more common use, and [Prometheus does refer to it that way](https://prometheus.io/docs/concepts/data_model/), as well. I would've assumed `timeseries` was the name of a database, or table, or something (today I learned).

Good suggestion from my point of view 👍🏻